WhaleSpotting/react-app/src/components/Navbar.tsx

```diff
@@ -38,25 +42,55 @@ export default function Navbar(): JSX.Element {
                         className={CheckCurrentPage("Reportsighting")}
                         onClick={() => HandleLinkClick("Reportsighting")}>Report Sighting</Link>
                 </div>
-                <div className="changing-nav-links">
+                <div className="changing-nav-links" >
                     <div hidden={loggedIn} >
-                        <Link to="/register"
-                            className={CheckCurrentPage("Register")}
-                            onClick={() => HandleLinkClick("Register")} >Register</Link>
-                        <Link to="/login"
-                            className={CheckCurrentPage("Login")}
-                            onClick={() => HandleLinkClick("Login")}>Login</Link>
+                        <div className="changing-nav-links-mobile" >
+                            <Link to="/register"
+                                className={CheckCurrentPage("Register")}
+                                onClick={() => HandleLinkClick("Register")} >Register</Link>
+                            <Link to="/login"
+                                className={CheckCurrentPage("Login")}
+                                onClick={() => HandleLinkClick("Login")}>Login</Link>
+                        </div>
                     </div>
                     <div hidden={!loggedIn}>
-                        <Link to="/Profile"
-                            className={CheckCurrentPage("Profile")}
-                            onClick={() => HandleLinkClick("Profile")}>Profile</Link>
-                        <Link to="/logout"
-                            className={CheckCurrentPage("Logout")}
-                            onClick={() => HandleLinkClick("Logout")}>Log Out</Link>
+                        <div className="changing-nav-links-mobile" >
+                            <Link to="/Profile"
+                                className={CheckCurrentPage("Profile")}
+                                onClick={() => HandleLinkClick("Profile")}>Profile</Link>
+                            <Link to="/logout"
+                                className={CheckCurrentPage("Logout")}
+                                onClick={() => HandleLinkClick("Logout")}>Log Out</Link>
+                        </div>
                     </div>
                 </div>
             </nav>
+            <div className="burger-button"
+                onClick={() => setBurgerState(!closeBurger)}>
+                <div className={closeBurger ? "opened-bar-1" : "closed-bar-1"}></div>
+                <div className={closeBurger ? "opened-bar-2" : "closed-bar-2"}></div>
+                <div className={closeBurger ? "opened-bar-3" : "closed-bar-3"}></div>
+            </div>
+            <div className={closeBurger ? "current-page-mobile" : "nav-bar"}>{currentPage == "Reportsighting" ? "Report Sighting" : currentPage}</div>
```

Comment:
Can we move this to before the burger button, as it will be shown before the burger button, also 
```suggestion
            <div className={closeBurger ? "current-page-mobile" : "nav-bar"}>{currentPage === "Reportsighting" ? "Report Sighting" : currentPage}</div>
```

Reply:
Done

Reply:
The equals change hasn't been done

Reply:
equals now done